lib/cdo/github.rb

```diff
@@ -92,6 +95,28 @@ def self.compare(base:, head:)
     response.commits.map(&:commit).map(&:message)
   end
 
+  # Octokit Documentation: http://octokit.github.io/octokit.rb/Octokit/Client/Commits.html#compare-instance_method
+  # @param base [String] The base branch to compare against.
+  # @param compare [String] The comparison branch to compare.
+  # @raise [Exception] From calling Octokit.compare.
+  # @return [Boolean] Whether compare is ahead of base, i.e., whether compare
+  #   has commits not in base.
+  def self.ahead?(base:, compare:)
+    response = Octokit.compare(REPO, base, compare)
+    response.ahead_by > 0
+  end
```

Comment:
Hmm. My default instinct is to remove dead code, esp. since we have no test over this.  Should be trivial to reconstruct from the `behind?` method below, too.  Let's cut it.